Formula/onefetch.rb

```diff
@@ -30,6 +30,6 @@ def install
     system "git", "config", "user.name", "BrewTestBot"
     system "git", "config", "user.email", "BrewTestBot@test.com"
     system "echo \"puts 'Hello, world'\" > main.rb && git add main.rb && git commit -m \"First commit\""
-    assert_match(/Language:.*Ruby/, shell_output("#{bin}/onefetch").chomp)
+    assert_match(/Ruby (100.0 %)/, shell_output("#{bin}/onefetch").chomp)
```

Comment:
Current failure is due to parentheses being seen as regex capture group.

Can just match on string literal. 
```suggestion
    assert_match "Ruby (100.0 %)", shell_output("#{bin}/onefetch").chomp
```

Reply:
Thanks a lot for the help @cho-m